config.go

```diff
@@ -36,6 +43,12 @@ func NewLoader(backends ...backend.Backend) *Loader {
 	return &l
 }
 
+// SetLoggerOutput is used for setting a writter for Confita's logger.
+// By default, it logs to ioutil.Discard.
+func (l *Loader) SetLoggerOutput(w io.Writer) {
+	l.log.SetOutput(w)
```

Comment:
I think it would be more flexible and simpler to simply export the Logger field and let people customize it if they want.
Not directly related, but this article is a good one to understand how to efficiently design a struct https://middlemost.com/object-lifecycle/